build/mvn

```diff
@@ -37,10 +37,10 @@ install_app() {
   local curl_opts=""
```

Comment:
Minor suggestion: Maybe change this to `local curl_opts="-L"` and update the block below to append options to this list? That way we don't have to duplicate the option.